### PR TITLE
[ADD] new relic: add for V13

### PR DESCRIPTION
**Pull request by Yenthe666**

### CODE REVIEW

**Comment by Yenthe666** [comment]
@jaredkipe here is the PR for 13. As you can see no code changes where needed but I restructured the `readme` and made it a bit more clear as it felt a bit confusing to me :)

**Comment by Yenthe666** [comment]
P.S: If you merge this in you might want to change your default branch to `13.0` too

File: README.rst [hunk 1]
```diff
@@ -70,17 +80,10 @@ Known Issues
 * Background tasks are not profiled.
 * Could probably use better transaction naming.
 
-
-Docker
-======
-
-Want to test this module out, but don't have gevent and newrelic installed?  I've got a Docker image for you!
-`hibou/odoo:9.0 <https://hub.docker.com/r/hibou/odoo/>`_
```
Comment (sswapnesh) [inline README.rst:78]: Maybe keep this and just change the version to 13.0 here

One can always change ODOO_VERSION at https://hub.docker.com/r/hibou/odoo/dockerfile (9.0 was already wrong referenced)

Comment (Yenthe666) [reply README.rst:78]: Finall call is for @jaredkipe on what he prefers I guess :)

Comment (jaredkipe) [reply README.rst:78]: Ultimately I'm fine with that, because we run a very different docker setup and build these days.

**Pull request merged**